core/types/crosslink_heartbeat.go

```diff
@@ -0,0 +1,9 @@
+package types
+
+type CrosslinkHeartbeat struct {
+	ShardID   uint32
+	BlockNum  uint64
```

Comment:
Rename to: LatestContinuousBlockNum     so it's not confusing to readers.

Reply:
fixed